analyses/stackanalyses/sa.go

```diff
@@ -0,0 +1,186 @@
+package stackanalyses
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"mime/multipart"
+	"net/http"
+	"os"
+	"time"
+
+	"github.com/rs/zerolog/log"
+
+	"github.com/fabric8-analytics/cli-tools/analyses/pypi"
+	"github.com/fabric8-analytics/cli-tools/utils"
+	"github.com/jpillora/backoff"
+)
+
+// SARequestType is a argtype of RequestServer func
+type SARequestType struct {
+	UserID          string
+	Host            string
+	ThreeScaleToken string
+	ShellPath       string
```

Comment:
What is the use of passing ShellPath to request?

Reply:
In Ext, we are explicitly setting shell path in VS Code settings. Just to maintain that practice, the idea is to enable user of having different shells.

Reply:
We don't allow the user to set shell path from vscode., we just propagate the same shell used by vscode to lsp due workaround the issue when launching vscode from gui launcher.

Reply:
Ack. Remove Shell Path flag and constant